python/pyiceberg/io/fsspec.py

```diff
@@ -150,9 +150,12 @@ def exists(self) -> bool:
         """Checks whether the location exists"""
         return self._fs.lexists(self.location)
 
-    def open(self) -> InputStream:
+    def open(self, seekable: bool = False) -> InputStream:
```

Comment:
I think we should default seekable to `True`

Reply:
Sure, done!